### DISCUSSION TITLE
Help - GitHub Foundations Exam

### DISCUSSION THREAD

**Discussion by FedeMontero** [community/community#136647] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Good morning,

I kindly ask for your advice regarding my GitHub Foundations certification because I don't know how to resolve my issue.

On August 7, 2024, I logged in at the scheduled time (6:00 PM Uruguay Standard Time) to take the certification exam. However, the Secure Browser never progressed beyond the initial screen. Before the exam, I conducted practice tests, checked my computer, and confirmed that everything was functioning properly.

Despite my efforts, I was unable to take the exam.

For the past two weeks, I’ve been calling PSIexams.com. At first, they told me not to worry, but when I called yesterday, they advised me to contact GitHub.

I would greatly appreciate any help, as I’m not sure what to do next.

Thank you very much.

Best regards.

**Comment by ebndev** [comment]
Hi @FedeMontero,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
